app/models/tube.rb

```diff
@@ -27,4 +27,12 @@ class Tube < ApplicationRecord
   def identifier
     barcode
   end
+
+  def position
+    nil
+  end
+
+  def labware_type
+    'tube'
```

Comment:
Could this be a constant? Or even better, there might be a way to get it from the class name, using `self`?